Cllog/Projects/CllogService/Features/EditFeature/Sources/Presentation/ThumbnailsView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ThumbnailsView.swift
+//  EditFeature
+//
+//  Created by seunghwan Lee on 3/9/25.
+//  Copyright © 2025 Supershy. All rights reserved.
+//
+
+import SwiftUI
+import AVKit
+
+/// 편집 섬네일 컷 (10장)
+struct ThumbnailsView: View {
+    let asset: AVAsset
+    let duration: Double
+    let thumbnailSize: CGSize
+    let thumbnailCount: Int
+    
+    init(asset: AVAsset, duration: Double, frameSize: CGSize) {
+        self.thumbnailCount = 10
+        self.asset = asset
+        self.duration = duration
+        self.thumbnailSize = CGSize(width: frameSize.width / CGFloat(thumbnailCount), height: frameSize.height)
+    }
+    
+    @State private var thumbnails: [UIImage] = []
+    
+    public var body: some View {
+        HStack(spacing: 0) {
+            ForEach(thumbnails.indices, id: \.self) { index in
+                Image(uiImage: thumbnails[index])
```

Comment:
```suggestion
            ForEach(thumbnails, id: \.self) { image in
                Image(uiImage: image)
```

이렇게 사용하지 않은 이유가 UIImage가 Hashable 채택을 안했었나 기억이 안나네ㅠ

Reply:
이미지보단 인덱스가 효율적일것 같아서!